W-19666248 HTTP Listener Updates (#1132)
mulesoft/docs-gateway: mule-gateway/modules/ROOT/pages/mule-gateway-config-autodiscovery-mule4.adoc

----

<1> Configure the `apiId` with the API ID that API Manager assigned to your API.

> <2> Set the `flowRef` element to point to the flow that you want to pair to the API in API Manager. The referenced flow must use an xref:connectors::http/http-listener-ref.adoc[HTTP Listener] component. Other connectors that use HTTP as an underlying transport are not supported for policy enforcement.

Author: ```suggestion
<2> Set the `flowRef` element to point to the flow that you want to pair to the API in API Manager. The referenced flow must use an xref:connectors::http/http-listener-ref.adoc[HTTP Listener] component. Other connectors that use HTTP as an underlying transport aren't supported for policy enforcement.
```
